apps/web/src/pages/integrations/components/multi-provider/CreateProviderInstanceSidebar.tsx

```diff
@@ -75,40 +76,66 @@ export function CreateProviderInstanceSidebar({
     }
   };
 
+  useEffect(() => {
+    if (!environments?.length) {
+      return;
+    }
+
+    reset({
+      name: provider.displayName,
+      environmentId: environments.find((env) => env.name === 'Development')?._id || '',
+    });
+  }, [environments, provider]);
+
```

Comment:
when the environment are loaded we should reset the state and set the default value